### PR TITLE
Add note about `vaadin.launch-browser`

### PR DESCRIPTION
**Pull request by tarekoraby**

### CODE REVIEW

File: articles/flow/integrations/spring/configuration.asciidoc [hunk 1]
```diff
@@ -13,6 +13,8 @@ Spring configuration properties have the same names, but are prefixed with `vaad
 
 == Special Configuration Parameters
 
+=== Configure the Scanning of Packages
+
 `blacklisted-packages` is a comma-separated string that can be used to blacklist packages from being scanned in v14 (npm mode) projects.
 The default set of packages does not cover everything that we should not be interested in.
```
Comment (dougjewsbury) [inline articles/flow/integrations/spring/configuration.asciidoc:19]: This is rather an uncomfortable double negative. Could we say:
"The default set of packages includes some packages that we are not interested in."?

File: articles/flow/integrations/spring/configuration.asciidoc [hunk 2]
```diff
@@ -37,6 +39,16 @@ vaadin.whitelisted-packages=com/foo/myapp/ui,com/foo/components
...
+
+.application.properties
+[source,properties]
+----
+vaadin.launch-browser = true
+----
+
 == Using Spring Boot Properties
 
 You can set properties for Spring Boot in your [filename]#application.properties`# file.
```
Comment (dougjewsbury) [inline articles/flow/integrations/spring/configuration.asciidoc:54]:
```suggestion
You can set properties for Spring Boot in your [filename]#application.properties# file.
```

File: articles/flow/integrations/spring/configuration.asciidoc (hunk 1 above)
Comment (tarekoraby) [reply articles/flow/integrations/spring/configuration.asciidoc]: The sentence is ambiguous indeed. I rewrote it. Hopefully, it's better now.

File: articles/flow/integrations/spring/configuration.asciidoc [hunk 3]
```diff
@@ -13,8 +13,11 @@ Spring configuration properties have the same names, but are prefixed with `vaad
 
 == Special Configuration Parameters
 
-`blacklisted-packages` is a comma-separated string that can be used to blacklist packages from being scanned in v14 (npm mode) projects.
-The default set of packages does not cover everything that we should not be interested in.
+=== Configure the Scanning of Packages
+
+To speed up the build of projects, Flow automatically excludes a number of packages (such as the ones belonging to the `java` and `springframework` packages) from being scanned.
```
Comment (joheriks) [inline articles/flow/integrations/spring/configuration.asciidoc:18]:
```suggestion
To decrease the startup time during development as well as the build time for the production bundle, Flow automatically excludes a number of packages (such as the ones belonging to the `java` and `springframework` packages) from being scanned for annotations.
```

File: articles/flow/integrations/spring/configuration.asciidoc [hunk 4]
```diff
@@ -13,8 +13,11 @@ Spring configuration properties have the same names, but are prefixed with `vaad
...
+The set of packages that Vaadin Flow excludes by default are defined in the [classname]#VaadinServletContextInitializer# class.
+You can extend this list using the `blacklisted-packages` property, which is a comma-separated string that can be used to blacklist packages from being scanned in npm-mode projects.
```
Comment (joheriks) [inline articles/flow/integrations/spring/configuration.asciidoc:20]:
```suggestion
You can extend this list using the `vaadin.blacklisted-packages` property, which is a comma-separated string that can be used to exclude packages from being scanned.
```

**Review by joheriks — CHANGES_REQUESTED** [review]

**Comment by joheriks** [comment]
Needs rebase.

**Pull request merged**